### ISSUE TITLE
Modify all notification properties from script/rules

### ISSUE THREAD

**Issue by emptymess** [dunst-project/dunst#899]
When using rules with messaging apps like Rochet.Chat, I would like to be able to capture specific notifications using wildcards and setting stack tags according to the trigger, e.g : 
```
[my-app]
    summary = "example-summary*"
    set_stack_tag = "%s"
```
`Notification > example-summary-foobar`
`Stack tag >> example-summary-foobar`

or even : 
```
[my-app]
    summary = "ex(am)ple-summary-(*)"
    set_stack_tag = "%1-%2"
```
`Notification > example-summary-foobar`
`Stack tag >> am-foobar`

This would allow modifying notifications dynamically with the help of just one rule and without scripting.

**Comment by fwSmit** [comment]
> This would allow modifying notifications dynamically with the help of just one rule and without scripting.

I'd say there's nothing wrong with scripting for these more advanced use cases. The problem is that I believe one can't modify the notification's stack tag or any other properties from a script.

**Comment by emptymess** [comment]
Well technically from script you could take the properties of a notification, process those properties, kill the processed notification from id and then spawn a corrected notification although it seems a bit overkill.

**Comment by fwSmit** [comment]
Yeah editing a notification's properties from the script seems like the right choice here. It's not something I'm planning to implement soon tho
